addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -20,30 +20,58 @@ sAnimations.registry.ProductWishlist = sAnimations.Class.extend(ProductConfigura
     },
     events: sAnimations.Class.events,
 
-    start: function () {
+    /**
+     * Init variables.
+     *
+     * @override
+     */
+    init: function (parent) {
+        this._super.apply(this, arguments);
+        this.wishlistProductIDs = [];
+    },
+    /**
+     * Get the current wishlist items.
+     *
+     * @override
+     * @returns {Deferred}
+     */
+    willStart: function () {
         var self = this;
         var def = this._super.apply(this, arguments);
-        if (this.editableMode) {
-            return def;
-        }
-
-        this.wishlistProductIDs = [];
 
         var wishDef = $.get('/shop/wishlist', {
```

Comment:
That RPC was not done in edit mode before.

Reply:
I didn't want to bother with an additional IF but I guess I'll add it.

Honestly this whole widget is pointless in edit mode, there should be a way to not initialize at all in that case?

Reply:
Yes, but this is like this for now. What you want will be done in my branch which depends on the CHM stuff.